modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl

```diff
@@ -39,10 +39,15 @@ FORSETI_COMMAND+=" --forseti_db $SQL_SERVER_LOCAL_ADDRESS/${cloudsql_db_name}?ch
 FORSETI_COMMAND+=" --config_file_path ${forseti_server_conf_path}"
 FORSETI_COMMAND+=" --services $FORSETI_SERVICES"
 
-CONFIG_VALIDATOR_COMMAND="${forseti_home}/external-dependencies/config-validator/ConfigValidatorRPCServer"
+CONFIG_VALIDATOR_COMMAND="$(which docker) run -p 50052:50052 --name config-validator"
```

Comment:
Add a `--rm` between `run` and `-p 50052:50052`

Reply:
Done